coriolis/tests/api/v1/test_minion_pools.py

```diff
@@ -0,0 +1,431 @@
+        if exception_raised:
+            self.assertRaisesRegex(
+                Exception,
+                exception_raised,
+                self.minion_pools._check_pool_numeric_values,
+                minimum_minions,
+                maximum_minions,
+                minion_max_idle_time
+            )
+        else:
+            self.assertEqual(
+                self.minion_pools._check_pool_numeric_values(
+                    minimum_minions, maximum_minions, minion_max_idle_time),
+                None)
+
+    @mock.patch.object(minion_pools.MinionPoolController,
+                       '_check_pool_retention_strategy')
+    @mock.patch.object(minion_pools.MinionPoolController,
+                       '_check_pool_numeric_values')
+    @ddt.file_data('data/minion_pools_validate_create_body.yml')
+    def test__validate_create_body(
+        self,
+        mock__check_pool_numeric_values,
+        mock__check_pool_retention_strategy,
+        config,
+        exception_raised,
+        expected_result
+    ):
+        expected_validation_api_method = config.get(
+            "expected_validation_api_method", None)
+        ctxt = {}
+        body = config["body"]
+        minion_pool = body["minion_pool"]
+        minion_pool["os_type"] = getattr(
+            constants, minion_pool["os_type"], None)
+        minion_pool["platform"] = getattr(
+            constants, minion_pool["platform"], None)
+        if 'minion_retention_strategy' in minion_pool:
+            minion_pool["minion_retention_strategy"] = \
+                getattr(
+                    constants, minion_pool["minion_retention_strategy"],
+                    constants.MINION_POOL_MACHINE_RETENTION_STRATEGY_DELETE)
+        minion_retention_strategy = minion_pool.get(
+            'minion_retention_strategy',
+            constants.MINION_POOL_MACHINE_RETENTION_STRATEGY_DELETE
+        )
+        minimum_minions = minion_pool.get('minimum_minions', 1)
+        maximum_minions = minion_pool.get('maximum_minions', 1)
+        minion_max_idle_time = minion_pool.get('minion_max_idle_time', 1)
+
+        if exception_raised:
+            self.assertRaisesRegex(
+                Exception,
+                exception_raised,
+                testutils.get_wrapped_function(
+                    self.minion_pools._validate_create_body),
+                self.minion_pools,
+                ctxt,
+                body
+            )
+        else:
+            with mock.patch.object(
+                endpoints_api.API,
+                expected_validation_api_method) as mock_validation_api_method:
+
+                result = testutils.get_wrapped_function(
+                    self.minion_pools._validate_create_body)(
+                        self.minion_pools,
+                        ctxt,
+                        body,
+                )
+
+                self.assertEqual(
+                    expected_result,
+                    list(result)
+                )
```

Comment:
Please replace this with the previously suggested statement:
```suggestion
                self.assertEqual(
                    result, tuple(expected_result))
```